dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -590,6 +590,31 @@ public void testUpdateWorkflowPath() throws ApiException {
         assertEquals("test workflow path should be the same as default workflow path, it is " + testpath, "/Dockstore.cwl", testpath);
     }
 
+    @Test
+    public void testAddingWorkflowForumUrl() throws ApiException {
+        // Set up webservice
+        ApiClient webClient = WorkflowIT.getWebClient(USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowsApi = new WorkflowsApi(webClient);
+
+        UsersApi usersApi = new UsersApi(webClient);
+        usersApi.getUser();
+
+        Workflow workflow = workflowsApi
+                .manualRegister(SourceControl.GITHUB.getFriendlyName(), "DockstoreTestUser2/test_lastmodified", "/Dockstore.cwl",
+                        "test-update-workflow", DescriptorLanguage.CWL.toString(),
+                        "/test.json");
+        
+        //update the forumUrl to hello.com
+        workflow.setForumUrl("hello.com");
+        workflowsApi.updateWorkflow(workflow.getId(), workflow);
+        workflowsApi.refresh(workflow.getId(), false);
+
+        //check the workflow's forumUrl is hello.com
+        final String updatedForumUrl = testingPostgres
+                .runSelectStatement("select forumurl from workflow where workflowname = 'test-update-workflow'", String.class);
+        assertEquals("forumUrl should be updated, it is " + updatedForumUrl, "hello.com", updatedForumUrl);
```

Comment:
For future, you don't have to say "it is ", the test runner will automatically say something like:
"expected: potato"
"actual: tomato"